content/cpp/concepts/destructors/destructors.md

```diff
@@ -0,0 +1,182 @@
+---
+Title: 'Destructors'
+Description: 'A destructor is a special method in a C++ class that is automatically called when an instance of that class is destroyed.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Classes'
+  - 'Objects'
+  - 'Destructors'
```

Comment:
'Destructors' is not a valid tag here, refer to the complete list of allowed tags [here](https://github.com/Codecademy/docs/blob/main/documentation/tags.md).

```suggestion
  - 'Memory'
```